sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/EncryptionSetIdentity.cs

```diff
@@ -51,7 +51,7 @@ public EncryptionSetIdentity()
         /// associated with the disk encryption set. The user identity
         /// dictionary key references will be ARM resource ids in the form:
         /// '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.</param>
-        public EncryptionSetIdentity(string type = default(string), string principalId = default(string), string tenantId = default(string), IDictionary<string, EncryptionSetIdentityUserAssignedIdentitiesValue> userAssignedIdentities = default(IDictionary<string, EncryptionSetIdentityUserAssignedIdentitiesValue>))
+        public EncryptionSetIdentity(string type = default(string), string principalId = default(string), string tenantId = default(string), IDictionary<string, UserAssignedIdentitiesValue> userAssignedIdentities = default(IDictionary<string, UserAssignedIdentitiesValue>))
```

Comment:
It looks like `EncryptionSetIdentityUserAssignedIdentitiesValue` was renamed as `UserAssignedIdentitiesValue`. Can you confirm?